### PR TITLE
Add vue

### PR DESCRIPTION
**Pull request by bitPogo**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,10 +4,15 @@ services:
     node:
         build:
             context: ./
+            dockerfile: ./Dockerfile
         image: wmde/wikibase-vuejs-components 
         volumes:
             - '~/.npm:/.npm'
+            - './../..:/src:ro' # grant read access to allow e.g. jest watch to detect changes
```
Comment (micgro42) [inline docker-compose.yml]: This going two directories up doesn't make that much sense here, as we are already in the root directory

**Review by micgro42 — COMMENTED** [review]
I think we should merge the docker config into a single commit. And the current docker config is not working for me.

Also, we shouldn't manually check in the dist/ files here.

**Comment by bitPogo** [comment]
> I think we should merge the docker config into a single commit.

I suggest we make a cleanup step anyways afterwards, where we squash and pull out those things...at the current state it's a bit tricky to do that, since I cannot simply amend things...

> And the current docker config is not working for me.

Could you provide some additional information? I cannot verify that, since it run on my local machine as well as on a virtual machine... 

> Also, we shouldn't manually check in the dist/ files here.

I guess you want to do that with something else, right? But at this stage of there is just the manual option or you we lost the possibility to verify that this works.

File: docker-compose.yml (hunk 1 above)
Comment (bitPogo) [reply docker-compose.yml]: Done

**Comment by micgro42** [comment]
These contributions have moved to Gerrit: https://gerrit.wikimedia.org/r/q/project:wikibase%252Fvuejs-components

**Pull request closed without merging**